pkgs/by-name/am/amber-lang/package.nix

```diff
@@ -36,14 +37,22 @@ rustPlatform.buildRustPackage rec {
     util-linux
   ];
 
+  checkFlags = [
+    "--skip=tests::extra::download"
+    "--skip=tests::formatter::all_exist"
+  ];
+
   postInstall = ''
     wrapProgram "$out/bin/amber" --prefix PATH : "${lib.makeBinPath [ bc ]}"
   '';
 
-  passthru.tests.run = runCommand "amber-lang-eval-test" { nativeBuildInputs = [ amber-lang ]; } ''
-    diff -U3 --color=auto <(amber -e 'echo "Hello, World"') <(echo 'Hello, World')
-    touch $out
-  '';
+  passthru = {
+    updateScript = nix-update-script { };
```

Comment:
It has been picked up by r-ryantm, but since it doesn't pass tests, r-ryantm didn't open PRs. See https://nixpkgs-update-logs.nix-community.org/amber-lang/

Reply:
I don't think keeping this hurts thou?

Reply:
If we know that updating via nix-update works well, preferring it is absolutely fine.